rm_manual:
  robot_type: "sentry"
  control_param:
    accel_x: 4.0
    accel_y: 5.0
    accel_angular: 10.0
    brake_multiple: 2
    expect_shoot_hz: 10
    safe_shoot_hz: 3
    safe_shoot_speed: 10
    gimbal_error_limit: 0.2
    rc_param:
      coefficient_x: 3.0
      coefficient_y: 3.0
      coefficient_angular: 6.0
      coefficient_yaw: 2.0
      coefficient_pitch: 2.0
  auto_move:
    chassis_speed: 1.6
    chassis_accel: 1.0
    pitch_speed: 0.8
    yaw_speed: 2.5
    start: 0.0
    end: 2.0
    collision_distance: 0.3
    calibration_speed: 0.2
    column: 1
  power_limit:
    safety_power: 30
    have_power_manager: false